doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -21,6 +21,47 @@ This specification is licensed under the
 [FIWARE Open Specification License]
 (http://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/FI-WARE_Open_Specification_Legal_Notice_%28essential_patents_license%29).
 
+## FIWARE NGSI information and data model introduction
+
+The FIWARE NGSI (Next Generation Service Interface) API defines 
+
+* a **data model** for context information, based on a simple information model using the notion of *context entities*.
+* a **context data interface** for exchanging information by means of query, subscription, and update operations.
```

Comment:
Not sure if from an user perspective is better from him/her to have one integrated interfaces that two sub-interfaces. To be discussed.